src/renderer/pages/player-page.js

```diff
@@ -227,6 +248,15 @@ function renderAudioMetadata (state) {
     ))
   }
 
+  // Audio metadata: disk & track-number
+  const count = ['track', 'disk']
+  count.forEach(key => {
+    const nrElem = renderTrack(common, key, key[0].toUpperCase() + key.substring(1))
```

Comment:
Here we can remove the third param, which is not needed anymore.
:)

Reply:
Oops, yeah sure.